pkg/controller/resources/route.go

```diff
@@ -131,7 +128,7 @@ func makeRoute(ci networkingv1alpha1.IngressAccessor, host string, index int, ru
 
 	route := &routev1.Route{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:            name,
+			Name:            host,
```

Comment:
As discussed in Slack, I this *feels* a little off as we'll create a new set of routes when we change the domain config for example. I don't have a strong opinion though. @bbrowning do you?

Reply:
Discussed this with @bbrowning, who was uncomfortable with just the hostname as a unique thing as well. I also realized that `$ns-$name-$index` isn't necessarily unique (ex: ns: test, name: test-service and ns: test-test, name service would result in the same thing).

Shall we use `route-${ci.GetUID()}-$index` instead? that should be guaranteed to be unique. Not exactly user-friendly but the user isn't supposed to mess with this anyway.

WDYT @nak3 @savitaashture ?

Reply:
It would be fine for uniqueness. But as reported https://github.com/openshift-knative/knative-openshift-ingress/pull/16, it would be more difficult to read the log message...

Reply:
Oh that we can fix differently. Commented on the relevant lines to add more context to our logs.